ethr-did/src/main/java/me/uport/sdk/ethrdid/EthrDID.kt

```diff
@@ -61,19 +61,19 @@ class EthrDID(
         return rawResult.substring(rawResult.length - 40).prepend0xPrefix()
     }
 
-    suspend fun changeOwner(newOwner: String): String {
+    suspend fun changeOwner(newOwner: String, txOptions: TransactionOptions? = null): String {
         val owner = lookupOwner()
 
         val encodedCall = EthereumDIDRegistry.ChangeOwner.encode(
             Solidity.Address(address.hexToBigInteger()),
             Solidity.Address(newOwner.hexToBigInteger())
         )
 
-        return signAndSendContractCall(owner, encodedCall)
+        return signAndSendContractCall(owner, encodedCall, txOptions)
```

Comment:
`txOptions` doesn't seem to be used in this method

Reply:
what do you mean by it isn't used?